cms/lib/xblock/authoring_mixin.py

```diff
@@ -39,7 +43,7 @@ def visibility_view(self, _context=None):
         """
         fragment = Fragment()
         from cms.djangoapps.contentstore.utils import reverse_course_url
-        fragment.add_content(self.system.render_template('visibility_editor.html', {
+        fragment.add_content(loader.render_mako_template('templates/visibility_editor.html', {
```

Comment:
It doesn't look like this pattern is being used in `edx-platform`? So to keep the changeset small, let's just use the service for now. We can do a mass switch later if needed.

Reply:
Ok, will do.

Reply:
Sorry no, I need to keep this change to `AuthoringMixin` as-is. The issue isn't with the `AuthoringMixin` itself, it's with the tests. When the `AuthoringMixin` is used in Studio, it's using the `PreviewModuleSystem`, which does have the `MakoService`, and so everything works fine.

However, [test_authoring_mixin.py](https://github.com/open-craft/edx-platform/blob/jill/bd-13-render_template/cms/lib/xblock/test/test_authoring_mixin.py#L56-L60) uses the `CachingDescriptorSystem`, and since I reverted b8c177d9d30cae70698818ad0ae830da53263009, the descriptor system doesn't contain the `MakoService`, and so the tests throw a `NoSuchServiceError`.

However, I was able to revert the other similar change made by b8c177d9d30cae70698818ad0ae830da53263009  to the `PollBlock`: that can just use the `MakoService` and leave its template in place.

Does that make sense?